lib/phoenix/code_reloader/server.ex

```diff
@@ -47,13 +47,14 @@ defmodule Phoenix.CodeReloader.Server do
 
   def handle_call({:reload!, endpoint}, from, state) do
     compilers = endpoint.config(:reloadable_compilers)
+    reloadable_apps = endpoint.config(:reloadable_apps)
```

Comment:
Can we generalize the logic here? Something like this:

    endpoint.config(:reloadable_apps) || default_reloadable_apps()

The `default_reloadable_apps()` will return all umbrella children name (for umbrella case) or a list with only the current app in regular cases. This way we don't need conditional branches later on when compiling. And setting the list to an empty list effectively disables reloading.

Reply:
yep this is a good point

Reply:
For now it is fine if we don't. If it becomes an issue we can discuss it.